netutils/vlan.py

```diff
@@ -1,12 +1,17 @@
 """Functions for working with VLANs."""
 
 import re
-
-from operator import itemgetter
+import typing as t
 from itertools import groupby
+from operator import itemgetter
 
 
-def vlanlist_to_config(vlan_list, first_line_len=48, other_line_len=44, min_grouping_size=3):
+def vlanlist_to_config(
+    vlan_list: t.List[int],
+    first_line_len: t.Optional[int] = 48,
+    other_line_len: t.Optional[int] = 44,
+    min_grouping_size: t.Optional[int] = 3,
```

Comment:
All of these probably shouldn't be optional, then we can also get rid of some of the ignores below